### ISSUE TITLE
Can the 2d data stacked with real images,then feed into CNN?

### ISSUE THREAD

**Issue by berisfu** [alok-ai-lab/DeepInsight#15]
Frankly speaking, if the imgs converted by the tabular table can be stacked with  real images or satellite imagery, it will be perfect.

**Comment by alok-ai-lab** [comment]
It may be possible to merge in two layers; i.e. the image produced by Deepinsight for tabular data, and the real image (grayscale) in two separate layers. However, the pixel locations of the produced tabular data and the real image may not coincide. It may work but we have not done such experiments.

On the other hand, if you produce an image by Deepinsight (say of size 120 x 120) and a real image (of size 120 x 120), you can merge in one layer of size 240 x 120 or 120 x 240. This can be further resized before giving to CNN based on the nets requirement, e.g. from 240x120 to 224x224 for ResNet.

**Comment by berisfu** [comment]
Because tabular data is naturally easy to use in traditional machine learning frameworks such as XGBOOST, LIGHTGBM, etc., CNN is not a must. Therefore, if the converted image from the tabular table is just only for CNN, it is actually not very convincing. CNN is not necessarily better than LIGHTGBM in fitting ability. But when it comes to combining table data and real image data, traditional machine learning is actually powerless. The most commonly used method is nothing more than adding the features in these tables to the FC layer, but the accuracy of the learned model generally does not exceed expectation. And the most terrible thing is that this method of adding to the FC layer is completely out of the question when it comes to learning similar to the UNET/FCN structure.
If  the method is proved to be effective for this scenarion, it will be a huge surprise.

**Comment by kaboroevich** [comment]
Closed due to inactivity - if your question is not answered, please feel free to reopen.
